packages/runtime/src/next/api-handler/index.ts

```diff
@@ -36,7 +37,7 @@ export function MakeswiftApiHandler(
     appOrigin = 'https://app.makeswift.com',
     apiOrigin = 'https://api.makeswift.com',
     getFonts,
-    unstable_siteVersions = false,
+    features,
```

Comment:
See my other comment on not wanting to have these nested under a `features` object